Lib/opcode.py

```diff
@@ -285,10 +285,17 @@ def jabs_op(name, op):
     ],
     "LOAD_ATTR": [
         "LOAD_ATTR_ADAPTIVE",
+        # LOAD_ATTR
         "LOAD_ATTR_INSTANCE_VALUE",
         "LOAD_ATTR_MODULE",
         "LOAD_ATTR_SLOT",
         "LOAD_ATTR_WITH_HINT",
+        # LOAD_METHOD
+        "LOAD_ATTR_METHOD_CLASS",
```

Comment:
Maybe lose the `METHOD`? They aren't necessarily methods.

The old `LOAD_METHOD` specializations are generally attributes of the class, whereas the old `LOAD_ATTR` specializations were specializations for attributes stored in the object's dict (whether materialized like `LOAD_ATTR_MODULE` or virtual like `LOAD_ATTR_INSTANCE_VALUE`)

Renaming them according to the specialization, not the old instruction name would be clearer, IMO.

Reply:
All `LOAD_ATTR_METHOD_*` are meant to signal that the instruction pushes `unbound meth, self` onto the stack rather than `NULL, bound meth`.  The only exception is `LOAD_ATTR_METHOD_CLASS` which indeed behaves almost like a `LOAD_ATTR_CLASS`. I plan to clean up that discrepancy in the very near future (and perhaps merge both `LOAD_ATTR_CLASS` and the current `LOAD_METHOD_CLASS`.